services/handlers/authentication.ts

```diff
@@ -1,86 +1,155 @@
-import logger from '../utilities/logger'
```

Comment:
Remove formatting changes. Does not relate to PR.

Reply:
removed formatting.